.gitignore

```diff
@@ -10,3 +10,11 @@ config.sh
 *.key.pub
 serverless-output.yml
 package-lock.json
+
+# Serverless directories
+.serverless
+.webpack
+
+npm-debug*
+
+config-dev.yml
```

Comment:
Might want to consider just supporting `.env` files. Then you either pump variables in or not - while avoiding all of the console warnings with serverless. Just a suggestion.

Reply:
good suggestion, going to make this adjustment.

Reply:
done via: https://serverless.com/plugins/serverless-dotenv-plugin/